src/Components/Locations.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from "react";
+import APICalls from "./APICalls";
+
+class Locations extends Component {
+  constructor() {
+    super();
+    this.state = {
+      locations: [],
+      showList: false,
+    };
+  }
+
+  getLocations = () => {
+    APICalls.getLocations().then(response => {
```

Comment:
Great job getting the array to have the right format!

 In the future, you'll probably want to make this a `forEach`, since you're not doing anything with the return value from `.map` (it returns an array, as I know you you know because you used the return value for `listItems` on line 47)

 Alternately, and probably better, is to `map` over the data, making each value an object in the format you want. Then you can simply `setState` that `locations` should be that array, and you've got it! The reason this is better is that the code is easier to follow that way, and it also avoids potential problems from doing multiple `setState` calls that use the previous state. Because `setState` runs asynchronously, this can result in unexpected behavior. (Google "problems setState async" and you'll find a lot of great reading on this potential bug!)